easybuild/easyconfigs/f/Fiji/Fiji-20240605-1417.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PackedBinary'
+
+name = 'Fiji'
+version = '20240605-1417'
```

Comment:
I think we should using this. Their github do tag releases:
https://github.com/fiji/fiji/tags
https://github.com/fiji/fiji/archive/refs/tags/fiji-2.16.0.tar.gz

so they do have actual version numbers